Apps/Admin/Client/Store/Dashboard/DashboardActions.cs

```diff
@@ -87,64 +97,110 @@ public LoadDependentsAction(int timeOffset)
     }
 
     /// <summary>
-    /// The action representing the initiation of recurring users load action.
+    /// The action representing a successful logged in user action.
     /// </summary>
-    public class LoadRecurringUsersAction
+    public class LoggedInSuccessAction : BaseSuccessAction<IDictionary<DateTime, int>>
```

Comment:
Typo: In this action, `LoggedIn` should be `LoggedInUsers` to match the related actions.